BUILD.bazel

```diff
@@ -1341,7 +1341,11 @@ genrule(
             tmpdir="redis.tmp" &&
```

Comment:
instead of this, maybe convert the whole thing to a shell script and then pipe the whole thing to dev null?

Reply:
I'm a little hesitant to do that in this PR for a few different reasons:

- My goal here was just to make the CI usable, not to refactor the Bazel `genrules`—that is something we should do down the road too, but it needs to happen for all them. Some are more complicated than this one, and in some cases there are also Bazel functions we should probably use (needs adding some dependencies) to e.g. copy files without having to write scripts.

- It would become potentially more brittle and less readable since `$(location ...)` would need to be passed as parameters.

- I only want to suppress progress messages and warnings (and only the ones that seem safe to suppress), so piping everything to `/dev/null` feels a little risky to me.

Reply:
maybe we can just use `./suppress_output` script to suppress it for success run and print out the error on failure?

Reply:
That still suppresses everything unless there's an actual error. Do you feel that's fine? It felt risky to me since I'm not sure what kinds of errors might be printed, but for example, seeing the linker warning about `tmpnam` being unsafe made me realize there's a flag to make it us `mkstemp` again, so it was actually beneficial I think.

Reply:
But if I print them when there is a warning then it would print exactly the same warnings I'm trying to suppress here though :(

I can try to simplify the sed calls, maybe that'll make them easier to understand?